status-models/src/main/java/com/telecominfraproject/wlan/status/equipment/report/models/ManagedNeighbourEquipmentInfo.java

```diff
@@ -59,28 +60,18 @@ public void setRadioInfo(RadioType radioType, NeighbourRadioInfo neighbourRadioI
 	public Integer getAverageSignalStrenght(RadioType radioType)
 	{
 	   NeighbourRadioInfo info = getRadioInfo(radioType);
+	   List<Integer> rssiList = new ArrayList<>();
 	   
-	   int runningTotal = 0;
-	   int numEntries = 0;
-	   
-	   if(info != null)
-	   {
-	      for(NeighbourBssidInfo singleBssidInfo :  info.getBssIds())
-	      {
-	         runningTotal += singleBssidInfo.getSignal();
-	         numEntries++;
+	   if(info != null && info.getBssIds() != null){
+	      for(NeighbourBssidInfo singleBssidInfo :  info.getBssIds()){
+	         rssiList.add(singleBssidInfo.getRssi());   
 	      }
 	   }
```

Comment:
how about this:
`
	   List<Integer> rssiList = new ArrayList<>();
	   if(info != null && info.getBssIds() != null){
           info.getBssIds().forEach(bssid -> rssiList.add(bssid.getRssi()));
        }
`